instaclustr/resource_bundle_user.go

```diff
@@ -0,0 +1,166 @@
+package instaclustr
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceBundleUser() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBundleUserCreate,
+		Read:   resourceBundleUserRead,
+		Update: resourceBundleUserUpdate,
+		Delete: resourceBundleUserDelete,
+
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"username": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"password": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"initial_permissions": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default: "none",
+				ForceNew: true,
+			},
+
+			"bundle_name": {
+				Type: 	  schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceBundleUserCreate(d *schema.ResourceData, meta interface{}) error {
+	cluster_id := d.Get("cluster_id").(string)
+	username := d.Get("username").(string)
+	bundle_name := d.Get("bundle_name").(string)
+
+	//todo: check if rest proxy or schema reg and then return nill, because those 2 bundles don't allow user creation
```

Comment:
Does this need to be updated?

Reply:
This validation should be done for both creation & deletion of users of schema_registry and rest_proxy. Will add the validation and update the readme soon.

Reply:
done

Reply:
updated the README as well.